examples/testlibusb.c

```diff
@@ -183,9 +183,9 @@ static void print_device(libusb_device *dev, libusb_device_handle *handle)
 		return;
 	}
 
-	printf("Dev (bus %u, device %u): %04X - %04X speed: %s\n",
-	       libusb_get_bus_number(dev), libusb_get_device_address(dev),
-	       desc.idVendor, desc.idProduct, speed);
+	printf("Dev (bus %u, device %u): %04X - %04X speed: %s driver: %s\n",
+		libusb_get_bus_number(dev), libusb_get_device_address(dev),
+		desc.idVendor, desc.idProduct, speed, libusb_get_driver_name(dev));
```

Comment:
regarding tabs vs spaces - here is a very interesting case:
the file uses tabs for aligning, but the consequent alignment of multiline statement/expression should use spaces, since the first line "refix" is a fixed-width `printf(`:
```suggestion
	printf("Dev (bus %u, device %u): %04X - %04X speed: %s driver: %s\n",
	       libusb_get_bus_number(dev), libusb_get_device_address(dev),
	       desc.idVendor, desc.idProduct, speed, libusb_get_driver_name(dev));
```
and the original quthor of these suggested that consequent lines should be aligned regardless of the "tab size" configuration in the viewer/IDE